src/main/java/org/topbraid/jenax/util/JenaUtil.java

```diff
@@ -317,7 +318,7 @@ public static Model createDefaultModel() {
 	 * @return a new memory graph
 	 */
 	public static Graph createMemoryGraph() {
-		return Factory.createDefaultGraph();
+		return GraphFactory.createDefaultGraph();
```

Comment:
The replacement for `org.apache.jena.graph.Factory` is `org.apache.jena.graph.GraphMemFactory`.

`GraphFactory.createDefaultGraph` is teh ARQ default implementation choice which can, in theory, be different (e.g. for some  tests).

`GraphFactory.createJenaDefaultGraph()` is `GraphMemFactory.createDefaultGraph()`